scripts/docker/Dockerfile.nvhpc

```diff
@@ -1,6 +1,17 @@
-ARG BASE=nvcr.io/nvidia/nvhpc:23.7-devel-cuda12.2-ubuntu20.04
+ARG BASE=nvcr.io/nvidia/nvhpc:23.7-devel-cuda12.2-ubuntu20.04@sha256:4f344e501d63c902a3f51c1783b88be5c9f5093641fc15b51b7d387f16d6d7bb
 FROM $BASE
 
+ARG ADDITIONAL_PACKAGES
+
+RUN apt-get update && apt-get install -y \
+        bc \
+        wget \
+        ccache \
```

Comment:
Did you mean to add these?
If so please briefly explain why.

Reply:
that was by mistake, good catch